modules/bmo-editing-a-baremetalhost-resource.adoc

```diff
@@ -0,0 +1,46 @@
+// This module is included in the following assemblies: 
+//
+// post_installation_configuration/bare-metal-configuration.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="editing-a-baremetalhost-resource_{context}"]
+= Editing a BareMetalHost resource
+
+After deploying a cluster, you might need to edit a node's `BareMetalHost` resource. For example, if you deploy a cluster with the {ai-full} and you need to add baseboard management controller credentials, or you want to move a node from one cluster to another without deprovisioning it, you must edit the node's `BareMetalHost` resource. To edit a node's `BareMetalHost` resource without deprovisioning the node, use the following procedure.
+
+.Prerequisites
+
+* Ensure the node is in the `Provisioned`, `ExternallyProvisioned` or `Available` state.
+
+.Procedure
+
+. Get the list of nodes:
++
+[source,terminal]
+----
+$ oc get bmh -n openshift-machine-api
+----
+
+. Before editing the node's `BareMetalHost` resource, detach the node from Ironic by running the following command:
++
+[source,terminal]
+----
+$ oc annotate baremetalhost <node_name> -n openshift-machine-api 'baremetalhost.metal3.io/detached=true' <1>
+----
+<1> Replace `<node_name>` with the name of the node.
+
+. Edit the  `BareMetalHost` resource by running the following command:
++
+[source,terminal]
+----
+$ oc edit bmh <node_name> -n openshift-machine-api <1>
+----
+<1> Replace `<node_name>` with the name of the node.
```

Comment:
I think the last 2 of these callouts can be deleted as the first example gives enough indication.